.github/workflows/build_test_deploy.yml

```diff
@@ -138,3 +138,25 @@ jobs:
           TF_VAR_app_feedback_link: ${{ secrets.FEEDBACK_LINK }}
           TF_VAR_enable_google_analytics: Yes
           TF_VAR_logit_sink_url: ${{ secrets.LOGIT_SINK_URL }}
+                    
+  cypress-tests:
+    environment: dev
```

Comment:
`needs: deploy-image`

will want to add this here to both of them so that it runs post-deploy rather than concurrent with the test step